bindings/c/test/mako/mako.cpp

```diff
@@ -875,6 +875,16 @@ int workerProcessMain(Arguments const& args, int worker_id, shared_memory::Acces
 		}
 	}
 
+	if (args.disable_client_bypass > 0) {
+		err = network::setOptionNothrow(FDB_NET_OPTION_DISABLE_CLIENT_BYPASS, args.disable_client_bypass);
```

Comment:
You should use network::setOptionNothrow(FDB_NET_OPTION_DISABLE_CLIENT_BYPASS) instead. Not sure if setting an option to an integer value works at all if it does not expect a value.

Reply:
It appeared to work, but I agree this is btter